src/index.html

```diff
@@ -2,11 +2,14 @@
 	<head>
 		<title>Order Splitter</title>
 		<meta charset="UTF-8">
+		<!-- inject:js -->
 		<script src="main.js"></script>
-		<script src="parsers.js"></script>
 		<script src="order.js"></script>
-		<script src="install.js"></script>
+		<script src="parsers.js"></script>
+		<!-- endinject -->
+		<!-- inject:css -->
 		<link rel="stylesheet" href="styles.css">
+		<!-- endinject -->
```

Comment:
This render-blocking issue was not introduced as a result of this pull request so I'd recommend opening a new issue to address this.